docker/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
 # other examples of installing gcloud in a docker image:
 #   - AoU (https://github.com/all-of-us/workbench/blob/master/api/src/dev/server/Dockerfile#L34)
 #   - Terra Jupyter (https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/Dockerfile#L74)
-ENV CLOUD_SDK_VERSION 411.0.0
+ENV CLOUD_SDK_VERSION 439.0.0
```

Comment:
thanks for getting to it so quickly Amy. I vote we just grab the latest.

Reply:
@yuhuyoyo now that I suggested that, I can't see how to do it.  AFAICT there is no download tagged 'latest'. Which probably explains why the version was pinned to start with.  (Do you know?).

Reply:
ah bummer. it does seem that it needs to be specified according to https://cloud.google.com/sdk/docs/install.

Reply:
I prefer an approach of pulling a specific version and then filing a Jira ticket to update every <N> months.
We had issues around CloudSDK in particular periodically breaking dsub/tests until we stopped getting latest.